action.yaml

```diff
@@ -0,0 +1,62 @@
+name: 'fluvio'
+description: 'Installs Fluvio CLI from source with required dependencies (e.g. Helm, Rust) and starts a Local Cluster. Used for developing fluvio.'
+inputs:
+  spus:  # number of SPUs
+    description: 'number of SPUs to create for local cluster'
+    required: false
+    default: "1"
+  cluster-type:
+    description: 'Type of cluster (i.e. local or kubernetes)'
+    required: true
+    default: 'local'
+  log-options:
```

Comment:
It might be easier to understand by calling it `rust-log` and mentioning that it is equivalent to `RUST_LOG` in the description.

Reply:
commit 168c946 renames.